core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -746,4 +756,36 @@ private void ensureExists(AlluxioURI path) throws IOException {
         toMap(worker -> worker.getNetAddress().getHost(), BlockWorkerInfo::getNetAddress,
             (worker1, worker2) -> worker1));
   }
+
+  /**
+   * Identifies whether a {@link URI} uri is an Alluxio on Zookeeper URI.
+   *
+   * @param uri the input uri
+   * @return whether the uri is Alluxio on Zookeeper URI
+   */
+  private static boolean isZookeeperUri(URI uri) {
+    // TODO(lu) make Alluxio URI capable of processing Alluxio on Zookeeper URI
+    // and expose methods to get URI type and Zookeeper addresses.
+    String authority = uri.getAuthority();
+    return authority != null && authority.contains(ZOOKEEPER_IDENTIFIER);
```

Comment:
I would use regexp to check if this is a zk address, otherwise this check is not safe.